proto/dwtools/abase/l0.test/Buffer.test.s

```diff
@@ -10029,39 +10029,508 @@ function bufferResize_( test )
 
 //
 
+function bufferBytesGet( test )
+{
+  /* BufferNode */
+
+  if( Config.interpreter === 'njs' )
+  {
+    test.case = 'from empty BufferNode';
+    var src = BufferNode.alloc( 5 );
+    var got = _.bufferBytesGet( src );
+    var expected = new U8x( [ 0, 0, 0, 0, 0 ] );
+    test.identical( got, expected );
+
+    test.case = 'from BufferNode (array)';
+    var src = BufferNode.from( [ 2, 4 ] );
+    var got = _.bufferBytesGet( src );
+    var expected = new U8x( [ 2, 4 ] );
+    test.identical( got, expected );
+
+    test.case = 'from BufferNode (identical symbols)';
+    var src = BufferNode.alloc( 5, 'a' );
+    var got = _.bufferBytesGet( src );
+    var expected = new U8x( [ 97, 97, 97, 97, 97 ] );
+    test.identical( got, expected );
+
+    test.case = 'from BufferNode (string)';
+    var src = BufferNode.from( 'string' );
+    var got = _.bufferBytesGet( src );
+    var expected = new U8x( [ 115, 116, 114, 105, 110, 103 ] );
+    test.identical( got, expected );
+
+    test.case = 'from BufferNode (string with base64)';
+    var src = BufferNode.from( 'string', 'base64' );
+    var got = _.bufferBytesGet( src );
+    var expected = new U8x( [ 178, 218, 226, 158 ] );
+    test.identical( got, expected );
+  }
+
+  test.case = 'from string';
+  var src = 'string';
+  var got = _.bufferBytesGet( src );
+  var expected = new U8x( [ 115, 116, 114, 105, 110, 103 ] );
+  test.identical( got, expected );
+
+  /* BufferRaw */
+
+  test.case = 'from empty BufferRaw';
+  var src = new BufferRaw( 5 );
+  var got = _.bufferBytesGet( src );
+  var expected = new U8x( [ 0, 0, 0, 0, 0, ] );
+  test.identical( got, expected );
+
+  test.case = 'from BufferRaw';
+  var src = new U8x( [ 1, 2, 3 ] ).buffer;
+  var got = _.bufferBytesGet( src );
+  var expected = new U8x( [ 1, 2, 3 ] );
+  test.identical( got, expected );
+
+
+  test.case = 'from empty BufferRawShared';
+  var src = new BufferRawShared( 5 );
+  var got = _.bufferBytesGet( src );
+  var expected = new U8x( [ 0, 0, 0, 0, 0, ] );
+  test.identical( got, expected );
+
+  test.case = 'from changed BufferRawShared';
+  var src = new BufferRawShared( 5 );
+  var view = new U8x( src );
+  view[ 1 ] = 2;
+  view[ 2 ] = 4;
+  var got = _.bufferBytesGet( view );
```

Comment:
Please, change `view` to `src`. `view` is typed buffer, it is representation of original raw buffer.